hooks/post_gen_project.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import os
+
+PROJECT_DIRECTORY = os.path.realpath(os.path.curdir)
+
+
+def remove_file(filepath):
+    os.remove(os.path.join(PROJECT_DIRECTORY, filepath))
+
+
+if __name__ == '__main__':
+    if '{{ cookiecutter.use_pypi_deployment_with_travis }}'.lower() == 'n':
```

Comment:
uh, how do you mean?
Sorry, I'm not following what's the issue here, could you please give more details?

If user replies `'n'`, this removes the file, if user replies `'y'` (the default), it doesn't do nothing.
The `if __name__ == '__main__'` block is never empty.

Thanks for reviewing! :+1: